docker/runtime/python-2.7/sqs-trigger/kubeless.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import sys#!/usr/bin/env python
+
+import sys
+import traceback
+import os
+import imp
+import json
+
+from multiprocessing import Process, Queue
+import prometheus_client as prom
+import boto3
+
+mod_name = os.getenv('MOD_NAME')
+func_handler = os.getenv('FUNC_HANDLER')
+queue_name = os.getenv('QUEUE_NAME')
+timeout = float(os.getenv('FUNC_TIMEOUT', 180))
+
+aws_access_key = os.getenv('AWS_ACCESS_KEY_ID')
```

Comment:
Probably needs to look for AWS_SESSION_TOKEN to support short-lived credentials, like the ones that come with EC2 instance roles, ECS tasks, and such

Reply:
yes, with kube2iam, we can actually load IAM roles in the pod. So we don't need to load the keys as env var.